src/Pages/InteractiveMap.js

```diff
@@ -47,48 +47,36 @@ const InteractiveMap = () => {
       <h3> See a Bug? Have an Idea?</h3>
       <p>
         {" "}
-        We would love to hear from you! Feel free to submit issue to us through
-        our GitHub - in our frontend repository for any visual or map issues,
-        and our backend if you're noticing any issues with our data.
+       While we’re winding down active development on this project and won’t be adding new features at this time, we’d love to hear your feedback! Feel free to submit issues to us through our GitHub— - in our frontend repository for any visual or map issues, and our backend if you're noticing any issues with our data.
       </p>
       <p>
         {" "}
         For a list of planned upcoming additions to this project, please visit
         our GitHub pages.
       </p>
-      <p>
-        If you want to get involved, you can join our Tuesday Chi Hack Night
-        breakout group.
-      </p>
-      <p>
-        Finally, if you have feedback for us on this project, you can always
-        reach out to us on Twitter!
-      </p>
-      <div className="btn-container">
-        <a
-          href="https://github.com/chihacknight/breakout-groups/issues/217"
-          target="_blank"
-          rel="noreferrer"
-          aria-label="Breakout group information link"
-        >
-          <button className="action-btn">Breakout group information</button>
-        </a>
-        <a
+
+         <a
           href="https://github.com/chihacknight/ghost-buses-frontend"
           target="_blank"
           rel="noreferrer"
-          aria-label="Frontend Repository link"
+          aria-label="Frontend Repository"
         >
           <button className="action-btn">Frontend Repository</button>
```

Comment:
Currently, these two buttons appear with the default styling (gray background, rectangle).

We should apply a CSS class to fix the styling (I can help with this!)